tests/testthat/test-carehomes-support.R

```diff
@@ -454,8 +459,9 @@ test_that("Can compute initial conditions", {
 
   initial_y <- mod$transform_variables(initial$state)
 
-  expect_equal(initial_y$N_tot3, sum(p$N_tot))
-  expect_equal(initial_y$N_tot2, sum(p$N_tot))
+  expect_equal(initial_y$N_tot_sero_2, sum(p$N_tot))
```

Comment:
should this one be N_tot_sero_1 (otherwise you have the exact same test 2 lines below

Reply:
Corrected, and also reordered for clarity